features/step_definitions/basic_steps.rb

```diff
@@ -12,6 +12,39 @@
     visit new_user_registration_path
 end
 
+When('I fill in {string} field with {string}') do |field, input|
+    fill_in field, with: input
```

Comment:
same step definition as below try to only use one.

Reply:
Done and done!